### PR TITLE
Set default to v1 commands

### PR DESCRIPTION
**Pull request by Liujingfang1**
- change `kubebuilder init` default to v1 `init`
- add `docs` command in v1 commands
- change version in `PROJECT` file from 2 to 1

### CODE REVIEW

File: cmd/kubebuilder/initproject/init.go [hunk 1]
```diff
@@ -56,15 +56,16 @@ kubebuilder init repo --domain mydomain
 		},
 	}
 
+	v0comment := "Works onlu with project-version v0, "
```
Comment (mengqiy) [inline cmd/kubebuilder/initproject/init.go]: typo: only

File: cmd/kubebuilder/main.go [hunk 2]
```diff
@@ -52,7 +52,7 @@ func main() {
 	initproject.AddInit(cmd)
 	version.AddVersion(cmd)
 
-	if util.IsNewVersion() {
+	if util.IsNewVersion() || util.IsProjectNotInitialized() {
 		v1.AddCmds(cmd)
 	} else {
 		v0.AddCmds(cmd)
```
Comment (mengqiy) [inline cmd/kubebuilder/main.go]: Please move func `CheckInstall` to a right place :)

File: cmd/kubebuilder/util/util.go [hunk 3]
```diff
@@ -180,3 +180,18 @@ func IsNewVersion() bool {
 func ProjectExist() bool {
 	return IsNewVersion()
 }
+
+func IsProjectNotInitialized() bool {
+	dirs := []string{
+		"cmd",
+		"hack",
```
Comment (mengqiy) [inline cmd/kubebuilder/util/util.go]: I'm confused what is the purpose of this function.
`hack` only exists in an old kb project.

Comment (Liujingfang1) [reply cmd/kubebuilder/util/util.go]: The function checks if there exists an initialized project inside one directory.
This function is used when adding the default commands to `kubebuilder`. We want the default to be v1 commands, even for an empty folder.
There is a `hack` package in the new KB project. 
```
tree hack
hack
└── boilerplate.go.txt
```

File: cmd/kubebuilder/initproject/init.go (hunk 1 above)
Comment (Liujingfang1) [reply cmd/kubebuilder/initproject/init.go]: my bad. Good catch.

File: cmd/kubebuilder/main.go (hunk 2 above)
Comment (Liujingfang1) [reply cmd/kubebuilder/main.go]: Do you mean `IsProjectNotInitialized`?

File: cmd/kubebuilder/initproject/init.go (hunk 1 above)
Comment (Liujingfang1) [reply cmd/kubebuilder/initproject/init.go]: done

File: cmd/kubebuilder/main.go (hunk 2 above)
Comment (Liujingfang1) [reply cmd/kubebuilder/main.go]: This is the current list of binaries being checked `"kubebuilder-gen", "client-gen", "deepcopy-gen", "gen-apidocs", "informer-gen",
			"openapi-gen", "kubebuilder", "conversion-gen", "defaulter-gen", "lister-gen"`

For v1 commands, the binaries need to be available are `"kubebuilder", "controller-gen"`. Are there any other ones?

Comment (Liujingfang1) [reply cmd/kubebuilder/main.go]: Talked with @droot, we will keep this check as it is.

**Pull request merged**